ucx.pc.in

```diff
@@ -9,3 +9,4 @@ Description: Unified Communication X Library
 Version: @VERSION@
 Cflags: -I${includedir}
 Libs: -L${libdir} -lucs -luct -lucp
+Libs.private: -lucm -Wl,--undefined=ucm_init -Wl,--undefined=ucs_init -Wl,--undefined=uct_init -Wl,--undefined=ucp_global_init @BFD_LIBS@ @BFD_LDFLAGS@ @BFD_DEPS@
```

Comment:
can pkg-config take "ucs" dependency from UCS pkg-config file, and take "ucm" dependency from ucs pkg-config?
and have separate pkg-config from ucs, ucs, ucp, ucm
these are the deps:
- ucp->{uct, ucs, ucm}
- uct->{ucs, ucm}
- ucs->{ucm}

Reply:
> can pkg-config take "ucs" dependency from UCS pkg-config file, and take "ucm" dependency from ucs pkg-

done